container/network.go

```diff
@@ -1,62 +1,67 @@
 package container
 
 import (
-	godocker "github.com/fsouza/go-dockerclient"
+	"context"
+
+	"github.com/docker/docker/api/types"
+	docker "github.com/docker/docker/client"
 )
 
 // CreateNetwork creates a Docker Network with a namespace
-func CreateNetwork(namespace []string) (network *godocker.Network, err error) {
-	network, err = FindNetwork(namespace)
-	if network != nil || err != nil {
+func CreateNetwork(namespace []string) (networkID string, err error) {
+	network, err := FindNetwork(namespace)
```

Comment:
I know you're still working on it but just as a reminder this should return a network object and wait for it if needed

Reply:
As the function from docker sdk `client.NetworkCreate` only return the networkID, I change the API.
I will revert the change of API and make this modification in another PR. Return the ID is actually enough, we never do anything with the full returned network or service.

Reply:
Actually, I would like to leave it like this. We really never use anything else than the ID of the create thing. Tell me if you prefer having this in an other PR.